cli/hooks.py

```diff
@@ -20,35 +22,106 @@ def chdir(dirname):
 
 class Hooks:
 
+    flag = False
+
     def __init__(self):
         self.utils = Utils()
+        self.whobj = WebHook()
 
     def run_hook(self, hookname, appdata, path, hook_input_metric):
         try:
             exit_code = 0
             function_execution_start_time = datetime.now()
+            defChannel = '#rogeros-deploy'
             execution_result = 'SUCCESS'
+            message = 'default message'
+            envr = 'NA'
+
+            temp = hook_input_metric.split(',')
+            action, app_name, config_name, envr, user = (temp[0],
```

Comment:
Could we match on the substring like (app_name , env etc) and then extract the values. Splitting based on "," and assuming the first element is action , second is app_name , will fail if the hook_input_metric changes, since the order in will the hoock input metric will be generated is different. This might cause wrong message.